File: clients/java3/common/src/org/openamq/AMQDisconnectedException.java
package org.openamq;

import org.apache.log4j.Logger;
import org.openamq.client.message.AbstractMessage;

/**
 * AMQ disconnected exception.
 */
public class AMQDisconnectedException extends AMQException
{
    public AMQDisconnectedException(String msg)
    {
        super(msg);
    }
}    

    

File: dotnet/xmscommon/AMQDisconnectedException.cs
using System;
using System.Text;

namespace OpenAMQ
{
    public class AMQDisconnectedException : AMQException
    {
        public AMQDisconnectedException(int errorCode, string message)
            : base(errorCode, message)
        {
        }
        
        public AMQDisconnectedException(string message)
            : base(message)
        {            
        }
    }
}
